STACpopulator/populator_base.py

```diff
@@ -142,23 +146,39 @@ def create_stac_collection(self) -> dict[str, Any]:
     def publish_stac_collection(self, collection_data: dict[str, Any]) -> None:
         post_stac_collection(self.stac_host, collection_data, self.update, session=self._session)
 
+    def configure_app_logging(self, log_debug) -> None:
+        """Configure the logger for the App."""
+        # generating the log file name
+        implementation_name = type(self).__name__
+        fname = f"{implementation_name}_log_{datetime.strftime(datetime.now(), '%Y%m%d-%H%M%S')}.jsonl"
```

Comment:
Consider `utcnow` with datetime format using `Z` or UTC timezone notation ?

Reply:
You mean something like `datetime.strftime(datetime.utcnow(), '%Y%m%dT%H%M%S')` ?

Reply:
`datetime.utcnow().isoformat() + "Z"` or add the `datetime.timezone` object to the date returned by `datetime.now()`, and then `.isoformat()` that TZ-aware datetime to get something like `"2024-02-20T01:22:33.456789+00:00"`.